src/main.ts

```diff
@@ -255,31 +253,33 @@ class Benchmark extends utils.Adapter {
 		if (!this.config.secondaryMode) {
 			if (obj.command === 'test') {
 				// run all tests on test command - do not await, we want to respond to message
-				this.runTests(allTests);
+				this.runTests(Object.keys(allTests));
 			} else if (allTests[obj.command]) {
-				const selectedTests:Record<string, any> = {};
-				selectedTests[obj.command] = allTests[obj.command];
-				this.runTests(selectedTests);
+				// this.runTests({
+				// 	[obj.command]: allTests[obj.command]
+				// });
+				this.runTests([obj.command]);
 			} else if (obj.command === 'requestedMonitoring') {
 				// we have received a requested monitoring
 				if (!this.requestedMonitoring[obj.from]) {
 					this.requestedMonitoring[obj.from] = {cpuStats: [], memStats: [], eventLoopLags: [], time: []};
 				}
 
+				const monitoring = this.requestedMonitoring[obj.from];
 				if (typeof obj.message === 'object' && Array.isArray(obj.message.cpuStats)) {
```

Comment:
I get anxious every time I see `typeof xxx === 'object'` because of `null` :)
Probably not a problem in this case though.